Index: glib2.0-2.29.18/gio/giomodule.c
===================================================================
--- glib2.0-2.29.18.orig/gio/giomodule.c	2011-08-30 06:09:51.000000000 +0200
+++ glib2.0-2.29.18/gio/giomodule.c	2011-09-02 08:25:45.715970977 +0200
@@ -765,6 +765,8 @@
 
       /* Then load the compiled in path */
       g_io_modules_scan_all_in_directory_with_scope (GIO_MODULE_DIR, scope);
+      /* Hard-coded fallback directory for pre-multiarch compatibility */
+      g_io_modules_scan_all_in_directory ("/usr/lib/gio/modules");
 
       g_io_module_scope_free (scope);
 
